.strigo/config.yml

```diff
@@ -1,45 +1,88 @@
-## Class exercises - simple configuration for a single class
+## Class lab resources and exercises - simple configuration for a single class
 ---
 classes:
-  - exercises:
-      # `file` is relative to .strigo directory
+  - lab_resources:
+      - display_name: "AWS EC2 Instance"
+        server_image_id: "ami-0abcdef1234567890"
+        platform_type: "lab"
+        image_platform: "linux"
+        cloud_provider: "aws"
+        view_interface: "Desktop"
+        instance_type: "t2.micro"
+      - display_name: "Web Resource"
+        web_resource_url: "https://example.com"
+        platform_type: "web"
+    exercises:
       - file: "../hello-world/hello.md"
         title: "Hello!"
       - file: "../hello-world/world.md"
         title: "World!"
 
-## Class exercises - configuration for multiple classes
+## Class lab resources - configuration for multiple classes
 # ---
 # classes:
-#   # `class_id` may be omitted if there is only one class
-#   # in this configuration file
 #   - class_id: "<your_class_id>"
+#     lab_resources:
+#       - display_name: "AWS EC2 Instance"
+#         server_image_id: "ami-0abcdef1234567890"
+#         platform_type: "lab"
```

Comment:
since platform type specifies these options:
    LAB: 'lab',
    WEB: 'web',
    CLOUD_ACCOUNT: 'cloud-account',
    CONTAINER: 'container',
    LIBRARY_VM: 'library-vm'
    
    I think we can remove it from the specs if we are not intending to include anything other than "lab".
    This might be a good opportunity to discuss how we incision the integration with the library-vm typed resources...

Reply:
Keeping for now just for the sake of "we already have it working" 🙂